js/App.js

```diff
@@ -0,0 +1,79 @@
+import { QuadraticGame } from './QuadraticGame.js'
+import { ExperimentController } from './ExperimentController.js'
+
```

Comment:
App.js isn't an appropriate name for this but I'll change that later - this would be the root page / view of an end-to-end experiment. The trials array would be received from another page / view like an admin form page.